README.md

```diff
@@ -169,5 +169,11 @@ By making a contribution to this project, I certify that:
 
 1. What about things like http_parser and libuv? Does it make sense to see about bringing each into the foundation as their own projects?
 2. What about all the other repos under iojs/* and joyent/*, which of those will move over to the foundation?
-3. The Node.js TC is working to introduce the notion of Release Candidates into the release cycle. It's not yet clear how this lines up with the semver release strategy used by io.js. This will need to be explored.
-4. The Node.js TC has recently adopted a priority tagging strategy (e.g P-1, P-2, P-3, P-4) to indicate the priority of a given issue or PR. It's not yet clear how and if this will fit into the larger project strategy.
+
+## Notes
+
+1. The existing Node.js Workflow (https://nodejs.org/documentation/workflow/) contains the notion of "Milestones" (https://nodejs.org/documentation/workflow/#index_md_flexible_workflow_only_two_strict_rules). These are essentially Long Term Support Release targets used as goalposts for landing specific edits. The "LTS Candidate" process described in the Long Term Release section above would largely replace the Milestone mechanism. When the LTS WG declares that a particular Major.Minor release is an LTS Candidate, a tag should be created identifying it as such. The PR or Issue can then be tagged so that Issues or PR's that should be landed or closed before cutting the LTS Release can be tracked.
```

Comment:
This seems fine, but they are also useful for majors.

> a tag should be created identifying it as such.

Do you mean a tag or a milestone?

I'd prefer to use milestones for such, as they are closeable.